### PR TITLE
Improve YAML examples - windows/win_package.py

### PR DESCRIPTION
**Pull request by Fale**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
lib/ansible/modules/windows

##### ANSIBLE VERSION
```
2.2
```

##### SUMMARY
Improve examples

@gundalow
@nitzmahone

### CODE REVIEW

File: lib/ansible/modules/windows/win_package.py [hunk 1]
```diff
@@ -77,23 +77,22 @@
...
-- name: Install the vc thingy
+- name: Install the vVisual C thingy
   win_package:
-    name="Microsoft Visual C thingy"
-    path="http://download.microsoft.com/download/1/6/B/16B06F60-3B20-4FF2-B699-5E9B7962F9AE/VSU_4/vcredist_x64.exe"
-    Product_Id="{CF2BEA3C-26EA-32F8-AA9B-331F7E34BA97}"
-    Arguments="/install /passive /norestart"
+    name: Microsoft Visual C thingy
+    path: http://download.microsoft.com/download/1/6/B/16B06F60-3B20-4FF2-B699-5E9B7962F9AE/VSU_4/vcredist_x64.exe
+    Product_Id: {CF2BEA3C-26EA-32F8-AA9B-331F7E34BA97}
```
Comment (nitzmahone) [inline lib/ansible/modules/windows/win_package.py]: These will need to remain quoted (either kind)- naked `{` at the beginning of a YAML value will be interpreted as a dictionary.

File: lib/ansible/modules/windows/win_package.py [hunk 2]
```diff
@@ -77,23 +77,22 @@
...
 
-# Install/uninstall an msi-based package
-- name: Install msi-based package (Remote Desktop Connection Manager)
+- name: Install Remote Desktop Connection Manager from msi
   win_package:
-    path: "https://download.microsoft.com/download/A/F/0/AF0071F3-B198-4A35-AA90-C68D103BDCCF/rdcman.msi"
-    product_id: "{0240359E-6A4C-4884-9E94-B397A02D893C}"
-- name: Uninstall msi-based package
+    path: https://download.microsoft.com/download/A/F/0/AF0071F3-B198-4A35-AA90-C68D103BDCCF/rdcman.msi
+    product_id: {0240359E-6A4C-4884-9E94-B397A02D893C}
```
Comment (nitzmahone) [inline lib/ansible/modules/windows/win_package.py]: Quote

File: lib/ansible/modules/windows/win_package.py [hunk 3]
```diff
@@ -77,23 +77,22 @@
...
+
+- name: Uninstall Remote Desktop Connection Manager installed from msi
   win_package:
-    path: "https://download.microsoft.com/download/A/F/0/AF0071F3-B198-4A35-AA90-C68D103BDCCF/rdcman.msi"
-    product_id: "{0240359E-6A4C-4884-9E94-B397A02D893C}"
+    path: https://download.microsoft.com/download/A/F/0/AF0071F3-B198-4A35-AA90-C68D103BDCCF/rdcman.msi
+    product_id: {0240359E-6A4C-4884-9E94-B397A02D893C}
```
Comment (nitzmahone) [inline lib/ansible/modules/windows/win_package.py]: Quote

File: lib/ansible/modules/windows/win_package.py [hunk 4]
```diff
@@ -77,23 +77,22 @@
 '''
 
 EXAMPLES = '''
-# Playbook example
-- name: Install the vc thingy
+- name: Install the vVisual C thingy
```
Comment (nitzmahone) [inline lib/ansible/modules/windows/win_package.py]: s/vVisual/Visual/

**Review by nitzmahone — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/windows/win_package.py (hunk 4 above)
Comment (Fale) [reply lib/ansible/modules/windows/win_package.py]: Thanks, fixed

File: lib/ansible/modules/windows/win_package.py (hunk 1 above)
Comment (Fale) [reply lib/ansible/modules/windows/win_package.py]: True, sorry about that

File: lib/ansible/modules/windows/win_package.py (hunk 2 above)
Comment (Fale) [reply lib/ansible/modules/windows/win_package.py]: True, sorry about that

File: lib/ansible/modules/windows/win_package.py (hunk 3 above)
Comment (Fale) [reply lib/ansible/modules/windows/win_package.py]: True, sorry about that

**Pull request merged**